exporters/newrelic/src/main/java/io/opentelemetry/exporters/newrelic/NewRelicSpanExporter.java

```diff
@@ -70,4 +76,91 @@ public ResultCode export(List<Span> openTracingSpans) {
 
   @Override
   public void shutdown() {}
+
+  /**
+   * Creates a new builder instance.
+   *
+   * @return a new instance builder for this exporter.
+   */
+  public static Builder newBuilder() {
+    return new Builder();
+  }
+
+  /**
+   * Builder utility for this exporter. At the very minimum, you need to provide your New Relic
+   * Insert API Key for this to work.
+   *
+   * @since 0.1.0
+   */
+  public static class Builder {
+
+    private Attributes commonAttributes = new Attributes();
+    private SpanBatchSender spanBatchSender;
+    private String apiKey;
+    private boolean enableAuditLogging = false;
+
+    /**
+     * A SpanBatchSender from the New Relic Telemetry SDK. This allows you to provide your own
+     * custom-built SpanBatchSender (for instance, if you need to enable proxies, etc).
+     *
+     * @param spanBatchSender the sender to use.
+     * @return this builder's instance
+     */
+    public Builder spanBatchSender(SpanBatchSender spanBatchSender) {
+      this.spanBatchSender = spanBatchSender;
+      return this;
+    }
+
+    /**
+     * Set your New Relic Insert Key.
+     *
+     * @param apiKey your New Relic Insert Key.
+     * @return this builder's instance
+     */
+    public Builder newRelicApiKey(String apiKey) {
```

Comment:
can we shorten this to just `insertKey` or `apiKey`?  The `newRelic` bit seems redundant.

Reply:
can do

Reply:
done